docs/components/ResourcesFooter.js

```diff
@@ -0,0 +1,128 @@
+// @flow strict
+import { type Node } from 'react';
+import { Box, Flex, Icon, Link, Heading, Text } from 'gestalt';
+import FigmaLogo from '../graphics/home-page/figma-logo.svg';
+import SlackLogo from '../graphics/home-page/slack-logo.svg';
+
+type LinkListProps = {|
+  items: Array<{|
+    title: string,
+    url: string,
+    a11yLabel?: string,
+  |}>,
+  heading: string,
+  isInternal?: boolean,
+  icon: 'figma' | 'slack' | 'eng',
+|};
+function LinkList({ items, heading, icon, isInternal = true }: LinkListProps): Node {
+  return (
+    <Flex alignItems="start" gap={2}>
+      <Box width="32px" display="flex">
+        {
+          {
```

Comment:
nit: defining an object literal inline like this isn't the most readable — consider defining outside the function body as a map